docs/marionette.functions.md

```diff
@@ -173,6 +173,9 @@ All arguments that are passed to the triggerMethod call are passed along to both
 
 `triggerMethod("foo", bar)` will call `onFoo: function(bar){...})`
 
+To trigger several events and corresponding methods split the events by the space. Example:
+* `triggerMethod("render ready")` fires the "onRender" ans the "onReady" functions
```

Comment:
actually can we use `triggerMethod('foo bar')`?  Triggering render manually is probably not a great idea.